src/app/rpmostree-builtin-status.c

```diff
@@ -655,6 +686,8 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy,
       rpmostree_print_kv ("LiveCommit", max_key_len, live_replaced);
       if (is_booted)
         g_print ("%s%s", get_bold_end (), get_red_end ());
+      if (!print_live_pkgdiff (live_replaced, diff_format, max_key_len, NULL, error))
```

Comment:
In the common case of livefs'ing a pending deployment, doesn't this result in printing out the diff twice? Could we optimize for that case and not print the livefs if live target == pending commit?

Reply:
Yeah...but that creates a dependence between the two status stanzas, we'd have to suppress that behavior if using e.g. `rpm-ostree status -b` etc.  A lot more work.